.github/workflows/ci.yml

```diff
@@ -21,6 +21,6 @@ jobs:
       - name: install
         run: yarn
       - name: build
-        run: yarn build
+        run: npm run build
```

Comment:
I had to change this because of following error:

 `$ npm run build && nx run-many --target=test --all
npm WARN lifecycle The node binary used for scripts is /tmp/yarn--1597882447464-0.20033904016625392/node but npm is using /opt/hostedtoolcache/node/12.18.3/x64/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.`

Unfortunately, I have no clear answer of what caused this issue. I can imagine one of the added packages ` semver`  or `lodash-es`.

Reply:
That's super weird 🤷‍♂️